Doxyfile

```diff
@@ -917,7 +917,7 @@ WARN_LOGFILE           = doxy_warnings.txt
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ./main ./components ./docs/SETUP.md ./docs/CONTRIBUTING.md ./docs/CODE_OF_CONDUCT.md
+INPUT                  = ./main ./components ./docs/SETUP.md ./docs/CONTRIBUTING.md ./docs/CODE_OF_CONDUCT.md ./docs/PORTING.md
```

Comment:
@Brycey92 I guess you didn't actually check the doxygen output, because the nice `\ref` and anchor you wrote did nothing!